core/src/com/cloud/agent/resource/virtualnetwork/VirtualRoutingResource.java

```diff
@@ -259,8 +260,18 @@ private Answer execute(GetDomRVersionCmd cmd) {
         return new GetDomRVersionAnswer(cmd, result.getDetails(), lines[0], lines[1]);
     }
 
+    public boolean configureHostParams(final Map<String, String> params) {
+        if (_params.get("router.aggregation.command.each.timeout") == null) {
```

Comment:
the idea is: if router.aggregation.command.each.timeout is already set in agent.properties, then ignore the propagation.